From 92ef7cafe6a45071e6a8a3910a24d0ab87a3d508 Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@cloudera.com>
Date: Thu, 5 Jan 2017 14:16:08 -0800
Subject: [PATCH 2131/2748] Excluding TestDistributedShell. DistributedShell
 is primarily for testing YARN and our customers
 don't use it. The exluded tests are flaky, likely
 because of timing issues.

Change-Id: Ic04f98881b0b8956d34475ddb359b55a4d884562
---
 cloudera/excludes.txt |    1 +
 1 file changed, 1 insertion(+)

diff --git a/cloudera/excludes.txt b/cloudera/excludes.txt
index ca6d7c1..8d70274 100644
--- a/cloudera/excludes.txt
+++ b/cloudera/excludes.txt
@@ -26,3 +26,4 @@ org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebApp.java
 org/apache/hadoop/yarn/server/timeline/webapp/TestTimelineWebServices.java
 org/apache/hadoop/yarn/server/applicationhistoryservice/TestFileSystemApplicationHistoryStore.java
 org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/TestContinuousScheduling.java
+org/apache/hadoop/yarn/applications/distributedshell/TestDistributedShell.java
-- 
1.7.9.5

